pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -251,10 +251,22 @@ def _get_backend_info(cls, backend: "_QiskIBMQBackend") -> BackendInfo:
         filtered_characterisation = {
             k: v for k, v in characterisation.items() if k in characterisation_keys
         }
+        # see below for references for config definitions
+        # quantum-computing.ibm.com/services/resources/docs/resources/manage/systems/:
+        # midcircuit-measurement/
+        # dynamic-circuits/feature-table
         supports_mid_measure = config.simulator or config.multi_meas_enabled
-        supports_fast_feedforward = False
+        supports_fast_feedforward = (
+            hasattr(config, "supported_features")
+            and "qasm3" in config.supported_features
```

Comment:
Does the qasm3 support need more changes in tket to being able to handle the conversion?

Reply:
I don't quite follow, I believe not though. This is less a PR for supporting qasm3, but more that, for some reason, what seems to denote that a device has fast feedforward support (or dynamic circuits) in Qiskit is that it can support jobs submitted with qasm3.

Reply:
Great